lib/datasets/california-housing.rb

```diff
@@ -0,0 +1,64 @@
+require "csv"
+
+require_relative "dataset"
+require "rubygems/package"
+require "zlib"
+
+module Datasets
+  class CaliforniaHousing < Dataset
+    Record = Struct.new(:longitude,
+                        :latitude,
+                        :housingMedianAge,
+                        :totalRooms,
+                        :totalBedrooms,
+                        :population,
+                        :households,
+                        :medianIncome,
+                        :medianHouseValue)
+
+    def initialize
+      super()
+      @metadata.id = "california-housing"
+      @metadata.name = "California Housing"
+      @metadata.url = "https://www.dcc.fc.up.pt/~ltorgo/Regression/cal_housing.html"
+      @metadata.licenses = ["Unknown"]
```

Comment:
Ah, this dataset doesn't provide license information...:  http://lib.stat.cmu.edu/datasets/

Could you use an empty array instead by indicating no license?

```suggestion
      @metadata.licenses = []
```

Reply:
Can use CC0

Reply:
Wow, really?
Where can we show the license information?

Reply:
Ah, I found an e-mail from the author. Thanks!